src/docfx/build/dependency/DependencyResolver.cs

```diff
@@ -48,7 +49,7 @@ public DependencyResolver(GitCommitProvider gitCommitProvider, Lazy<XrefMap> xre
             return (error, link, file);
         }
 
-        public (Error error, string href, string display, Document file) ResolveXref(string href, Document relativeTo)
+        public (Error error, string href, string display, Document file) ResolveXref(string href, Document relativeTo, List<Document> callStack)
```

Comment:
>callStack [](start = 134, length = 9)

make it a default value `null` and assign it a value if it's not set, so the caller no need change code anymore?